backend/src/core/interfaces/campaign.interface.ts

```diff
@@ -123,4 +124,6 @@ export interface DuplicateCampaignDetails {
  *          updated_at:
  *            type: string
  *            format: date-time
+ *          error_code:
```

Comment:
Couple things:
- `CampaignInterface` is dead code; created an issue here
- Noticed we're not including optional properties here; from [Swagger docs](https://swagger.io/docs/specification/data-models/data-types/#:~:text=By%20default%2C%20all%20object%20properties%20are%20optional.%20You%20can%20specify%20the%20required%20properties%20in%20the%20required%20list%3A): By default, all object properties are optional. You can specify the required properties in the required list:

Probably not much point to fixing, but just flagging up